Assets/Scripts/InputManager/IPlayerInputManager.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace HarmonyQuest.Input

```

Comment:
What's our naming convention for namespaces? Mitch and I just used game name + folder path under the scripts folder. Whatever we decide on, it should be consistent.

Reply:
Yeah u rite